README.md

```diff
@@ -1,39 +1,56 @@
-# Helix
+<h1 align="center" style="font-size: 55px">Helix</h1>
+<p align="center" style="font-size: 30px">A kakoune / neovim inspired editor, written in Rust.</p>
 
+<div align="center" style="display:inline">
```

Comment:
I don't particularly like the screenshots, they don't use the default theme and the shine distracts away from the editor. We should show it as it looks out of the box. (I think the current screenshot looks "nice" enough but that's because I like my bitnap fonts)

Reply:
I understand not liking the acyclic, but I think using a theme that is appealing to the majority would help with first impressions(if you care).
The [Onedark theme](https://cdn.discordapp.com/attachments/542248938015555604/893330202799779880/unknown.png) is included and seems like a more popular choice.

It's seem like you want to show off features rather than looks, which is hard to do in a screenshot, maybe a few gifs would be a better choice? Like what [onivim](https://v2.onivim.io/) has on their website.

Up to you really, can change it back if you want.

Reply:
> using a theme that is appealing to the majority would help with first impressions

I think this a very subjective thing. Editors are usually presented as they are out of the box, and given that I've written both the editor and the theme, that's the theme that's default. See [Kakoune](https://kakoune.org/) for very plain screenshots that do a great job of showcasing basic features.

Reply:
> I think this a very subjective thing.

Whether a theme is good or not is subjective, however the popularity of a theme is not. One dark for example, has almost 7 million downloads on vscode, there is atleast something quantifiable about that.

As for Kakoune, it looks at least 30 years out of date(the gifs are okay). Which is frustrating because I'm sure the features are good. There's no need to compromise like that, you can have both functionality and aesthetics. Again [onivim](https://onivim.github.io/blog/) does this well.